### PR TITLE
Add support for `pi0w2`

### LINKED ISSUE #1626: Can't detect wifi hardware on Raspberry Pi Zero W 2
I'm trying to get openhabian running on the new Pi 0 W2. However, when the setup runs it can't detect the wifi hardware of the board. As a result, I couldn't connect to the internet and It could not start hotspot as well.

### PR DESCRIPTION
**Pull request by ecdye**
Fixes #1626

Signed-off-by: Ethan Dye <mrtops03@gmail.com>

### CODE REVIEW

File: functions/packages.bash [hunk 1]
```diff
@@ -393,7 +393,7 @@ knxd_setup() {
 ##    miflora_setup()
 ##
 miflora_setup() {
-  if ! is_pizerow && ! is_pithree && ! is_pithreeplus && ! is_pifour; then
+  if ! is_pi_wlan; then
```
[the remark this answers is not in the document]
Comment (ecdye) [reply functions/packages.bash:396]: Well, the pi's with wlan are the only ones with bt AFAIK. It should be equivalent.

[the remark this answers is not in the document]
Comment (mstormi) [reply functions/packages.bash:396]: maybe add anaother is_pi_bt() = is_pi_wlan() function or at least put this as a comment in the code

**Comment by ecdye** [comment]
Done, @mstormi can we try and get this in before I release 1.7 today?

**Comment by mstormi** [comment]
Of course.
FYI I renamed the "stable release" menu to just "release" but did not make a PR out of it

**Pull request merged**